### PR TITLE
Init sc update

### PR DESCRIPTION
**Pull request by Eitu33**

### CODE REVIEW

**Review by Thykof — APPROVED** [review]
LGTM

File: assembly/std/index.ts [hunk 1]
```diff
@@ -79,19 +79,39 @@ export function localExecution(
...
+ * Get the bytecode of the current address
+ *
+ * @return {StaticArray<u8>} bytecode
+ */
+export function callerHasWriteAccess(): bool {
+  return env.callerHasWriteAccess();
+}
+
+/**
+ * Get the bytecode of the current address
```
Comment (AurelienFT) [inline assembly/std/index.ts:107]: Wrong copy/paste

File: assembly/std/index.ts [hunk 2]
```diff
@@ -79,19 +79,39 @@ export function localExecution(
...
  * @return {StaticArray<u8>} bytecode
  */
-export function getBytecodeFor(targetAddress: Address): Array<StaticArray<u8>> {
-  return derKeys(env.getBytecodeFor(targetAddress.toByteString()));
+export function getBytecodeFor(address: Address): StaticArray<u8> {
+  return env.getBytecodeFor(address.toByteString());
+}
+
+/**
+ * Get the bytecode of the current address
```
Comment (AurelienFT) [inline assembly/std/index.ts:98]: Wrong copy/paste

File: assembly/env/env.ts [hunk 3]
```diff
@@ -1,3 +1,5 @@
+import {Address} from '../std';
```
Comment (AurelienFT) [inline assembly/env/env.ts:1]: Should be removed

**Review by AurelienFT — APPROVED** [review]

**Pull request merged**